tests/utapi/awsNodeSdk.js

```diff
@@ -0,0 +1,375 @@
+const async = require('async');
```

Comment:
I feel like this isn't wrong but a bit much. Both with the mock UTAPI server with express and recreating a bunch of events to send to UTAPI. Part of my concern here is too much testing as well as potentially duplicate testing.
It seems to me our unit testing and functional testing overlap here as well as what would probably be included in UTAPI v2. 

I don't really want to block the PR to come up with some sort of test plan or Venn Diagram of what is tested where but I don't want to blindly review all this code.